ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -19,16 +19,28 @@
  * {@link Router#getBlob(String, GetBlobOptions, Callback)} call.
  */
 public class GetBlobOptions {
+  public static final GetBlobOptions DEFAULT_OPTIONS = new GetBlobOptions(OperationType.All, null);
```

Comment:
Since this is a read-only object and can be shared, isn't it better to have it as a `static` object?